site/package-lock.json

```diff
@@ -6595,25 +6595,24 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
I'm seeing these flip-flops resolved - I suspect I'm related to this, since I'm flipping one way and you're flopping another.   Not an issue with this PR, but let's double check next week on our local set ups to get it aligned.